### PR TITLE
[#300] Add the child device topics to c8y bridge

### PR DESCRIPTION
**Pull request by didier-wenzek**
## Proposed changes

* Forward to c8y the child-devices related topics.

## Types of changes

What types of changes does your code introduce to `thin-edge.io`?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement (general improvements like code refactorings that doesn't explicitly fix a bug or add any new functionality)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Paste Link to the issue
#300 

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
- [ ] I ran `cargo fmt` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I used `cargo clippy` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: tedge/src/cli/connect/bridge_config_c8y.rs [hunk 1]
```diff
@@ -50,10 +50,10 @@ impl From<BridgeConfigC8yParams> for BridgeConfig {
...
                 r#"s/ut/# out 2 c8y/ """#.into(),
                 // Static templates
-                r#"s/us out 2 c8y/ """#.into(),
-                r#"t/us out 2 c8y/ """#.into(),
-                r#"q/us out 2 c8y/ """#.into(),
-                r#"c/us out 2 c8y/ """#.into(),
+                r#"s/us/# out 2 c8y/ """#.into(),
+                r#"t/us/# out 2 c8y/ """#.into(),
+                r#"q/us/# out 2 c8y/ """#.into(),
+                r#"c/us/# out 2 c8y/ """#.into(),
```
Comment (rina23q) [inline tedge/src/cli/connect/bridge_config_c8y.rs:56]: The parent device still needs to publish `s/us`, `t/us`, `q/us`, and `c/us`. I think if we configure like this, we will miss the bridge for those topics for the parent device, won't we?
So, I think we still need to keep the configuration without `/#` for those topics.

**Review by rina23q — COMMENTED** [review]
I found we actually missed one more bridge configuration to support subscribing custom template. SUBSCRIBE `s/dc/myCommom_10` (Given my_Common_10 is the name of custom template). 

Do you have Cumulocity MQTT Cheat Sheet? If not, I'll share it with you.

File: tedge/src/cli/connect/bridge_config_c8y.rs (hunk 1 above)
Comment (didier-wenzek) [reply tedge/src/cli/connect/bridge_config_c8y.rs:56]: The pattern `s/us/#` match both `s/us` and `s/us/test123`.

[The multi-level wildcard represents the parent and any number of child levels](http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html#_Toc385349376)

Comment (rina23q) [reply tedge/src/cli/connect/bridge_config_c8y.rs:56]: It's written in the specs! I didn't know that. Thank you!

**Review by rina23q — APPROVED** [review]

**Pull request merged**